src/Pool.sol

```diff
@@ -66,6 +66,8 @@ abstract contract Pool {
     /// @notice Maximum number of receivers of a single sender.
     /// Limits costs of changes in sender's configuration.
     uint32 public constant SENDER_WEIGHTS_COUNT_MAX = 100;
+    /// @notice Maximum value of drips fraction
+    uint32 public constant DRIPS_FRACTION_MAX = 10000;
```

Comment:
Maybe we should just have on constant for fraction operations?
Currently `SENDER_WEIGHTS_SUM_MAX` has the same value. 

I am also wondering if we want a higher precision in general here. 

Consider the following case: Let say a project receives more the an 1m DAI.
Let say the want to give aways everything above the 1m in drips.

```
Project receives: 1.234.567 DAI
Wants to keep 1m
1-(1000000/1234567)=0.1899994087 (drip factor)
```

I would like to use maybe 10^27 here.

Reply:
> Maybe we should just have on constant for fraction operations?

I wouldn't mix them together, they have different consequences and `SENDER_WEIGHTS_SUM_MAX` is low for a reason. The fact that they're equal is a confusing coincidence.

> Let say the want to give aways everything above the 1m in drips.

That's not a use case for drips, you need to declare dripped fraction BEFORE funds are streamed to you, otherwise you could cheat your supporters on drips.

> I would like to use maybe 10^27 here.

I've been thinking about what scale should drips fraction use. The first idea was to just slap `type(uint128).max` and call it a day. But the dripped fraction is declared before you receive anything, so laser precision doesn't seem that useful. The other aspect is that numbers capped at 10^4 are very easy to work with for humans on low level, e.g. in tests or in etherscan. On high level the UI will probably ask the users for percentage on scale 0-100 anyway and multiply it to reach the desired scale. I'm OK with expanding the drips fraction scale, but I don't see the benefits of pushing it very far.